scripts/visualise_results.py

```diff
@@ -8,11 +9,22 @@
 
 def plot_data(df):
     plt.style.use('seaborn-poster')
-    df.plot.line()
+
+    if (
+        0 < int(os.getenv("NUMBER_OF_DAYS", 0)) <= 45
```

Comment:
`NUMBER_OF_DAYS` is already parsed in `parse_environment_variables` and could be passed as an argument to `plot_data`.